server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -118,7 +118,7 @@
     //TODO this needs to be cleaned up: _timestamp and _ttl are not supported anymore, _field_names, _seq_no, _version and _source are
     //also missing, not sure if on purpose. See IndicesModule#getMetadataMappers
     private static final String[] SORTED_META_FIELDS = new String[]{
-        "_id", IgnoredFieldMapper.NAME, "_index", "_routing", "_size", "_timestamp", "_ttl", "_type"
+        "_id", IgnoredFieldMapper.NAME, "_index", "_routing", "_size", "_timestamp", "_ttl", "_type", "_nested_path"
```

Comment:
I think you need to preserve the sort order ?

Reply:
++, will fix.  I had a look at fixing the TODO above, but it's not all that straightforward unfortunately.